src/main/java/io/aiven/kafka/connect/http/config/HttpSinkConfig.java

```diff
@@ -150,6 +154,18 @@ public boolean visible(final String name, final Map<String, Object> parsedConfig
             HTTP_HEADERS_CONTENT_TYPE_CONFIG
         );
 
+        configDef.define(
+                HTTP_HEADERS_ADDITIONAL,
+                ConfigDef.Type.LIST,
+                Collections.EMPTY_LIST,
+                ConfigDef.Importance.LOW,
+                "Additional headers to forward in the http request in the format header:value separated by a comma",
+                CONNECTION_GROUP,
+                groupCounter++,
+                ConfigDef.Width.MEDIUM,
+                HTTP_HEADERS_ADDITIONAL
+        );
```

Comment:
I think we should make use of the `Validator` in here and make sure the `http.headers.additional` values are following the right format:

- Comma separated
- Pairs of key value separated with a `:`
- No key is repeated

At the moment if a key is repeated the application will throw an exception only when the additional headers are retrieved from the config (not when the config is loaded). Additionally, if a key-value pair is malformed, this is simply ignored through the `.filter(h -> h.length > 1)` call.

Could you use the `define` variant that takes a `ConfigDef.Validator` checking the previously mentioned points?

Reply:
Additionally, could you add a test for this under the `HttpSinkConfigValidationTest` file?

Reply:
Added KeyValuePairListValidator for this and respective tests